corehq/apps/analytics/signals.py

```diff
@@ -132,4 +132,4 @@ def track_user_login(sender, request, user, **kwargs):
             return
 
         meta = get_meta(request)
-        track_user_sign_in_on_hubspot.delay(couch_user, request.COOKIES, meta)
+        track_user_sign_in_on_hubspot(couch_user, request.COOKIES, meta, request.path)
```

Comment:
For some reason that only works sometimes for me. I haven't tracked down
why though.

On Wed, Dec 23, 2015 at 11:01 AM Farid Rener <notifications@github.com>
wrote:

> In corehq/apps/analytics/signals.py
> <https://github.com/dimagi/commcare-hq/pull/9697#discussion_r48356329>:
>
> > @@ -132,4 +132,4 @@ def track_user_login(sender, request, user, **kwargs):
> >              return
> >
> >          meta = get_meta(request)
> > -        track_user_sign_in_on_hubspot.delay(couch_user, request.COOKIES, meta)
> > +        track_user_sign_in_on_hubspot(couch_user, request.COOKIES, meta, request.path)
>
> [image: :+1:] if you add CELERY_ALWAYS_EAGER= True to your localsettings
> I don't think you need to do that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9697/files#r48356329>.
>